resources/assets/js/features/search/components/Result/_Modals/SubscribeModal.js

```diff
@@ -0,0 +1,268 @@
+                                                    </div>
+                                                    <select name="luggage" value={luggage || ''} className="form-control" id="luggage" onChange={this.luggageChange}>
+                                                        <option value="">{translate('search_result.not_important')}</option>
+                                                        <option value="0">{translate('search_result.luggage_size_0')}</option>
+                                                        <option value="1">{translate('search_result.luggage_size_1')}</option>
+                                                        <option value="2">{translate('search_result.luggage_size_2')}</option>
+                                                    </select>
+                                                </div>
+                                            </div>
+                                        </div>
+                                        <div className="col-md-6 pr-4">
+                                            <div className="filter__prop">
+                                                <div className="filter__prop-name subscribe-modal-name">{translate('search_result.filter.price')}</div>
+                                                <div className="filter__prop-control">
+                                                    <div className="filter__prop-name subscribe-modal-name">
+                                                        {translate('search_result.filter.price_range', {start: price[0], end: price[1]})}
+                                                    </div>
+                                                    <Range
+                                                        min={0}
+                                                        max={2000}
+                                                        step={10}
+                                                        allowCross={false}
+                                                        value={price}
+                                                        onChange={(price) => this.setState({price})}
+                                                        pushable
+                                                    />
+                                                </div>
+                                            </div>
+                                            <div className="filter__prop">
+                                                <div className="filter__prop-control">
+                                                    <div className="filter__prop-name subscribe-modal-name">
+                                                        {translate('search_result.free_seats')}
+                                                    </div>
+                                                    <select name="seats" value={seats || ''} className="form-control" id="seats" onChange={this.seatsChange}>
```

Comment:
I think we already had similar dropdown somewhere, ensure that you don't have code duplication, you can or create method in service that will return such dropdowns or create component that will receive callback and default value text from props